CMSIS/NN/Include/arm_nnfunctions.h

```diff
@@ -78,12 +78,15 @@
    *
    * - ARM_MATH_DSP:
    *
-   * Define macro ARM_MATH_DSP, If the silicon supports DSP instructions.
+   * Define macro ARM_MATH_DSP, If the silicon supports DSP instructions(DSP extension).
    *
    * - ARM_MATH_MVEI:
    *
    * Define macro ARM_MATH_MVEI, If the silicon supports M-Profile Vector Extension.
-   *
+
+   * - ARM_MATH_AUTOVECTORIZE
```

Comment:
This flag is not related to ASM.
I need to document the meaning of this flag in arm_math.h
But today it is not yet applied very consistently in the DSP library.